crates/symbolicator/src/services/bitcode.rs

```diff
@@ -229,17 +229,24 @@ impl BitcodeService {
         uuid: DebugId,
         scope: Scope,
         sources: Arc<[SourceConfig]>,
-    ) -> Result<Option<BcSymbolMapHandle>, Error> {
+    ) -> Option<BcSymbolMapHandle> {
         // First find the PList.
         let find_plist = self
             .fetch_file_from_all_sources(uuid, AuxDifKind::UuidMap, scope.clone(), sources.clone())
-            .await?;
+            .await;
         let plist_handle = match find_plist {
             Some(handle) => handle,
-            None => return Ok(None),
+            None => return None,
         };
```

Comment:
This looks like it could be `let plist_handle = find_plist?;`.

Reply:
totally, i'm surprised clippy didn't pick that up